{
  "stats": {
    "suites": 0,
    "tests": 1,
    "passes": 0,
    "pending": 0,
    "failures": 1,
    "start": "2023-08-11T09:40:51.600Z",
    "end": "2023-08-11T09:40:52.622Z",
    "duration": 1022,
    "testsRegistered": 1,
    "passPercent": 0,
    "pendingPercent": 0,
    "other": 0,
    "hasOther": false,
    "skipped": 0,
    "hasSkipped": false
  },
  "results": [
    {
      "uuid": "ea783d25-fe9b-4841-be75-38f13e0bdbf2",
      "title": "",
      "fullFile": "cypress\\e2e\\bdd.cy copy.js",
      "file": "cypress\\e2e\\bdd.cy copy.js",
      "beforeHooks": [],
      "afterHooks": [],
      "tests": [
        {
          "title": "An uncaught error was detected outside of a test",
          "fullTitle": "An uncaught error was detected outside of a test",
          "timedOut": null,
          "duration": 427,
          "state": "failed",
          "speed": null,
          "pass": false,
          "fail": true,
          "pending": false,
          "context": "[\n  {\n    \"title\": \"cypress-mochawesome-reporter-videos\",\n    \"value\": \"cypress\\\\e2e\\\\bdd.cy copy.js\"\n  },\n  {\n    \"title\": \"cypress-mochawesome-reporter-screenshots\",\n    \"value\": [\n      [\n        {\n          \"title\": \"Failed screenshot\",\n          \"value\": \"\\\\bdd.cy copy.js\\\\An uncaught error was detected outside of a test (failed).png\"\n        }\n      ],\n      [\n        {\n          \"title\": \"Failed screenshot\",\n          \"value\": \"\\\\bdd.cy copy.js\\\\An uncaught error was detected outside of a test (failed) (attempt 2).png\"\n        }\n      ]\n    ]\n  }\n]",
          "code": "throw err;",
          "err": {
            "message": "TypeError: The following error originated from your test code, not from Cypress.\n\n  > \"\" is not a function\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.",
            "estack": "TypeError: The following error originated from your test code, not from Cypress.\n\n  > \"\" is not a function\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.\n    at 1.C:/Users/AB/Desktop/cypress/qa-cypress-project/node_modules/@babel/runtime/helpers/interopRequireDefault.js (cypress/e2e/bdd.cy copy.js:3:0)\n    at o (node_modules/browser-pack/_prelude.js:1:0)\n    at r (node_modules/browser-pack/_prelude.js:1:0)\n    at eval (node_modules/browser-pack/_prelude.js:1:0)\n    at eval (<anonymous>)",
            "diff": null
          },
          "uuid": "98b1603a-7e36-4de4-85dc-eae5ee657030",
          "parentUUID": "ea783d25-fe9b-4841-be75-38f13e0bdbf2",
          "isHook": false,
          "skipped": false
        }
      ],
      "suites": [],
      "passes": [],
      "failures": [
        "98b1603a-7e36-4de4-85dc-eae5ee657030"
      ],
      "pending": [],
      "skipped": [],
      "duration": 427,
      "root": true,
      "rootEmpty": false,
      "_timeout": 2000
    }
  ],
  "meta": {
    "mocha": {
      "version": "7.0.1"
    },
    "mochawesome": {
      "options": {
        "quiet": false,
        "reportFilename": "mochawesome",
        "saveHtml": false,
        "saveJson": true,
        "consoleReporter": "spec",
        "useInlineDiffs": false,
        "code": true
      },
      "version": "7.1.3"
    },
    "marge": {
      "options": {
        "overwrite": false,
        "html": false,
        "json": true,
        "reportDir": "cypress\\reports\\html\\.jsons"
      },
      "version": "6.2.0"
    }
  }
}